website/blog/2023-12-14-release-1.6.md

```diff
@@ -0,0 +1,253 @@
+---
+title: Release Notes - Podman Desktop 1.6
+description: Podman Desktop 1.6 has been released!
+slug: podman-desktop-release-1.6
+authors: slemeur
+tags: [podman-desktop, release, kubernetes, openshift, onboarding, compose, extensions, settings]
+hide_table_of_contents: false
+image: /img/blog/podman-desktop-release-1.6/SantaSeal.png
+---
+
+import ReactPlayer from 'react-player'
+
+Podman Desktop 1.6 Release! 🎉
+
+![Podman-desktop-1-6-hero](img/podman-desktop-release-1.6/SantaSeal.png)
+
+This release introduces:
+
+- **Minikube featured extension**: Minikube extension to create local Kubernetes clusters in containers.
+- **Podman 4.8.2**: [Podman 4.8.2](https://github.com/containers/podman/releases) is now included in Windows and Mac installers.
```

Comment:
Just a side note, we are still talking about podman-remote, right? Since on Windows the podman in the VM is right now 4.8.1 and on Mac it is 4.7.2. Just to be clear with users how this works.

Reply:
Yeah - but this will be updated once the VM images are getting available. I'm not really convince we should confuse the users either :(